### PR TITLE
layout-catalog

### PR DESCRIPTION
**Pull request by Gabai62**
- [DEMO LINK](https://Gabai62.github.io/layout_catalog/)
- [TEST REPORT LINK](https://Gabai62.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,379 @@
...
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops&computer"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+            Laptops & Computers
+            </a>
```
Comment (loralevitska) [inline src/index.html:40]:
```suggestion
            >
              Laptops & Computers
            </a>
```

File: src/styles/blocks/container.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
+.container {
+  padding-top: 50px;
+  padding-bottom: 72px;
```
Comment (loralevitska) [inline src/styles/blocks/container.scss:3]: Combine with `padding` shorthand property

File: src/styles/blocks/product-card.scss [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+    transform: scale(1.2);
+  }
+
+  &:hover &__title {
+    color: #34568b;
+  }
+
+  &__code {
+    margin-bottom: 0;
+    margin-top: 4px;
```
Comment (loralevitska) [inline src/styles/blocks/product-card.scss:37]: Combine with`margin` shorthand property. Fix all cases if present

File: src/styles/blocks/stars.scss [hunk 4]
```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
```
Comment (loralevitska) [inline src/styles/blocks/stars.scss:9]: Combine into `background` shorthand property

File: src/styles/blocks/stars.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    }
+  }
+
+  &--1 .stars__star:first-child,
+  &--2 .stars__star:nth-child(-n+2),
+  &--3 .stars__star:nth-child(-n+3),
+  &--4 .stars__star:nth-child(-n+4),
+  &--5 .stars__star:nth-child(-n+5) {
+    background-image: url(../images/star-active.svg);
+  }
```
Comment (loralevitska) [inline src/styles/blocks/stars.scss:23]: Rewrite it with loop 
image

File: src/styles/main.scss [hunk 6]
```diff
@@ -1,3 +1,20 @@
...
+
+@import "./blocks/header.scss";
+
+@import "./blocks/product-card.scss";
+
+@import "./blocks/stars.scss";
+
+@import "./blocks/container.scss";
+
+@import "./blocks/nav.scss";
```
Comment (loralevitska) [inline src/styles/main.scss:11]:
```suggestion
@import "./utils/variables.scss";
@import "./blocks/header.scss";
@import "./blocks/product-card.scss";
@import "./blocks/stars.scss";
@import "./blocks/container.scss";
@import "./blocks/nav.scss";
```

**Review by loralevitska — CHANGES_REQUESTED** [review]
Good job! There are things that should be fixed:

**Review by maxim2310 — APPROVED** [review]
Good work!!!

**Pull request closed without merging**